### ISSUE TITLE
[a11y] 11.1.1.1a Non-text content (control elements)

### ISSUE THREAD

**Issue by tschiebel** [owncloud/android#4360]
audit: https://infinite.owncloud.com/s/QtwuiGHikwFjozV

https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf
https://github.com/BIK-BITV/BIK-App-Test/tree/main/Pr%C3%BCfschritte/de

### Personal Space 

- [ ] Several control elements (buttons) have been detected on the page,
that do not have an alternative text.
![image](https://github.com/owncloud/android/assets/132444655/b0ae191e-9944-41d7-8392-db037e8be74d)

- [ ] Although the "Manage accounts" button already has an alternative text,
it is not very clear and does not help users to navigate through the app.
through the app. To ensure that the button is accessible to all users and enables effective
accessible to all users and enables effective navigation, it is best to simply
simply use "Manage accounts" as the alternative text. 
![image](https://github.com/owncloud/android/assets/132444655/1d3c2375-c7cf-4c62-a6a2-327bf75f04fc)

### Menu

- [ ] The button with the logo does not have an alternative text. 

### Share

- [ ] Several control elements (buttons) have been detected on the page,
that have no alternative text.
![image](https://github.com/owncloud/android/assets/132444655/c4d5a13f-ced8-4706-a8db-3e796469b7f3)

- [ ] The buttons with the "+" symbol do not have a meaningful alternative text
alternative text, as screen readers can only recognize the "+" sign.

- [ ] The "Back" button has an alternative text, but this is confusing as "Up" is displayed.
confusing, as it reads "Up". It would be best if
use texts such as "Back" or "Back to previous page" instead.
instead.
![image](https://github.com/owncloud/android/assets/132444655/4b17c925-753d-4046-8a42-0b7e19aa1db5)

### Spaces

- [ ] The "Menu" button does not have an alternative text.
![image](https://github.com/owncloud/android/assets/132444655/ae6d0afa-004d-4e20-9a83-5daa4b7d58e9)

- [ ] "Manage accounts" button - See comment on page 1.

### Manage accounts

- [ ] Several control elements (buttons) have been detected on the page,
that have no alternative text. 
![image](https://github.com/owncloud/android/assets/132444655/af8861fa-4fcb-4e6f-9606-d185a8e5745e)

- [ ] The "Back" button has an alternative text, but this is confusing as "Up" is displayed.
confusing as it reads "Up". It would be best if
use texts such as "Back" or "Back to previous page" instead.
instead.
![image](https://github.com/owncloud/android/assets/132444655/7e0a4681-4a52-4514-8e68-100bad95ae2c)

### Image Viewer

- [ ] The "Back" button has an alternative text, but this is confusing,
as "To top" is displayed. It would be best to use texts like
"Back" or "Back to previous page" instead.
![image](https://github.com/owncloud/android/assets/132444655/11971c69-8405-47c2-adc4-7bf7730c578b)

**Comment by jesmrec** [comment]
- Translatable strings
